### PR TITLE
added a section for parameter sensitivities calculation

### PR DESCRIPTION
**Pull request by wibraun**

### CODE REVIEW

File: UsersGuide/source/parametersensitivity.rst [hunk 1]
```diff
@@ -0,0 +1,94 @@
+Parameter Sensitivities with OpenModelica
+==============================
```
Comment (sjoelund) [inline UsersGuide/source/parametersensitivity.rst]: Wrong number of = signs for the topic

File: UsersGuide/source/parametersensitivity.rst [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+*Note: this is a very short preliminary description which soon will be
+considerably improved, since this a rather new feature and will
+continuous improved.*
+
+*Note: A OpenModelica version newer than openmodelica_1.10.0~dev-598-g33dd447
+is needed.*
+
+
+Background
+--------------------------------------------------------
```
Comment (sjoelund) [inline UsersGuide/source/parametersensitivity.rst]: Wrong number of - signs for "Background"

**Comment by wibraun** [comment]
@sjoelund Okay, fixed.

File: UsersGuide/source/parametersensitivity.rst [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+This section describes the use of OpenModelica to compute parameter
+sensitivities using forward sensitivity analysis together with
+sundials/idas solver.
+
+*Note: this is a very short preliminary description which soon will be
+considerably improved, since this a rather new feature and will
+continuous improved.*
+
+*Note: A OpenModelica version newer than openmodelica_1.10.0~dev-598-g33dd447
```
Comment (sjoelund) [inline UsersGuide/source/parametersensitivity.rst]: I guess this should rather say: this feature was introduced in OpenModelica v1.10 (the development versions can be skipped for when features are added)

**Comment by wibraun** [comment]
@sjoelund fixed version.

**Pull request merged**